### PR TITLE
upgrade to 5.4.3

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**

### CODE REVIEW

**Review by akabanovs — CHANGES_REQUESTED** [review]
ipython. >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0
qdarkstyle>=3.0.2,<3.2.0
qtconsole>=5.4.2,<5.5.0
spyder-kernels>=2.4.3,<2.5.0

Does it need ptyprocess? Can't find it in the setup.py
Does it need python.app ?
Not sure about menuinst (don't see it in setup.py)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,15 +56,15 @@ requirements:
...
+   - qtconsole >=5.4.2,<5.5.0
    - qtpy >=2.1.0
    - rtree >=0.9.7
    - setuptools >=49.6.0
    - sphinx >=0.6.6
-   - spyder-kernels >=2.4.2,<2.5.0
+   - spyder-kernels >=2.4.3,<2.5.0
    - textdistance >=4.2.0
    - three-merge >=0.1.1
    - watchdog >=0.10.3
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:70]:
```
    # This is here to workaround an inconsistency with flake8
    # See: https://github.com/conda-forge/flake8-feedstock/issues/52
   - importlib-metadata <4.3 # [py<38]
```
Can be removed as we do not build for py37

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:70]: removed

**Review by Arishamays1 — APPROVED** [review]
LGTM, but I would also suggest before merging, adding a summary of your changes to the description!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -56,21 +56,18 @@ requirements:
    - python-lsp-server >=1.7.1,<1.8.0
```
[the remark this answers is not in the document]
Comment (ELundby45) [reply recipe/meta.yaml:56]: Thank you very much for the heads up. A new build will be available shortly.

[the remark this answers is not in the document]
Comment (ccordoba12) [reply recipe/meta.yaml:56]: Thanks a lot for your prompt help @ELundby45!

**Pull request merged**